flexmeasures/cli/data_add.py

```diff
@@ -837,6 +837,13 @@ def create_forecasts(
     default=1,
     help="Round-trip efficiency (e.g. 85% or 0.85) to use for the schedule. Defaults to 100% (no losses).",
 )
+@click.option(
+    "--as-job",
+    is_flag=True,
+    help="Whether to queue a forecasting job instead of computing directly."
+    " Useful to run locally and create forecasts on a remote server. In that case, just point the redis db in your"
+    " config settings to that of the remote server. To process the job, run a worker to process the forecasting queue. Defaults to False.",
```

Comment:
Might be worthwhile to point out that running a local worker would result in getting the forecasts locally, while running a worker on the remote server would result in getting the forecasts on the remote server.

Reply:
I feel that computing locally has two additional requirements to work correctly:

- use the same SQL database as the server
- use the same FlexMeasures version as the server

In the end, I'd rather move this idea of using jobs on a local machine to the documentation, as it gets more intricate.